packages/lib/src/index.ts

```diff
@@ -1,3 +1,4 @@
+import fs from 'fs';
```

Comment:
```suggestion
```

I think we don't need this at all, `FileSystem` was imported here just to re-export it.